hail/src/test/scala/is/hail/expr/ir/OrderingSuite.scala

```diff
@@ -41,6 +42,63 @@ class OrderingSuite extends HailSuite {
     fb.resultWithIndex()(0, r)
   }
 
+  @Test def testMissingNonequalComparisons() {
```

Comment:
This test might be more complete if you compared the values to themselves, since the missing values being unequal case will only arise if the values being compared are identical up until the first set of fields where both are missing (not sure how likely this is to come up; we're only running 10 tests). If you have a test comparing values to themselves, you're at least forcing the missingness logic to come up once per test.

Reply:
Good idea!

Reply:
Actually, without needing to test deep missingness behavior, it's even easier.